packages/pf4-component-mapper/src/files/wizard.d.ts

```diff
@@ -52,6 +52,15 @@ export interface SubstepOfObject {
   title?: ReactNode;
 }
 
+export interface StepTemplateProps {
+  title?: ReactNode;
+  formFields: ReactNode;
+  showTitles?: boolean;
+  showTitle?: boolean;
```

Comment:
What is the difference between `showTitle` and `showTitles`? Maybe we could use different names?

Reply:
ShowTitles is on wizard, ShowTitle is on step

Reply:
Maybe rename the `showTitle` to something like show header? The difference is so tiny that I had to read it 3 times to notice the plural.

Reply:
It doesn't matter. showTitles is set in wizard definition. showTitle in wizard step.

Renaming it would be a breaking change.

Reply:
Ah. Well, that is a bummer. I was not aware it was used already. Nothing we can to at this point.